sycl/ReleaseNotes.md

```diff
@@ -13,6 +13,7 @@ codes in the same translation unit [31608c24] [f5126d28]
 `-fsycl-targets` [5e86a41d]
 
 ### SYCL Library
+- Made use of immediate command lists the default on Intel® Data Center GPU Max Series on Linux [9e5b2ed1] (https://www.intel.com/content/www/us/en/developer/articles/guide/level-zero-immediate-command-lists.html)
```

Comment:
Please remove the hash. We just need to add a link to doc and little explanation to it. 
Also, this is not a "New feature". This is just a documentation, so please move to the "Documentation" section.

Reply:
The commit id references the commit made to make immediate command lists the default. Any reason it should be left out?

Reply:
Got it, thanks. So this is not just adding some documentation - it is adding new feature + documentation. As the feature was made after March'23 release, this PR should be abandoned because this change goes to the next open-source release, and release notes for the next open source release will have it.
And regarding the upcoming closed-source release, if this feature were added to the upcoming closed-source release, please send this wording to TCEs, so they can include it to the release notes for the upcoming closed source release.

Reply:
When is the next open-source release and when will the RN for that be created?

Reply:
Answered internally.